src/rules/noUnreachableRule.ts

```diff
@@ -99,4 +100,8 @@ class NoUnreachableWalker extends Lint.RuleWalker {
         super.visitThrowStatement(node);
         this.hasReturned = true;
     }
+
+    public visitTypeAliasDeclaration(node: ts.TypeAliasDeclaration) {
```

Comment:
I don't think we actually need to add these lines - `visitNode` gets called on all nodes by default